go.mod

```diff
@@ -112,6 +112,7 @@ require (
 	go4.org v0.0.0-20190313082347-94abd6928b1d // indirect
 	golang.org/x/sync v0.0.0-20190423024810-112230192c58 // indirect
 	golang.org/x/sys v0.0.0-20190626221950-04f50cda93cb
+	google.golang.org/appengine v1.4.0 // indirect
```

Comment:
I have no idea why this dep started showing up but it is a heisen-dep, it shows up on build and disappears with tidy.

Reply:
Yeah, it's a bug in go and I don't feel like reporting it.

Reply:
Same.